src/components/AddressSearch.js

```diff
@@ -161,7 +165,7 @@ const AddressSearch = (props) => {
                         label: props.label,
                         containerStyles: props.containerStyles,
                         errorText: props.errorText,
-                        hint: props.hint,
+                        hint: displayListViewBorder ? undefined : props.hint,
```

Comment:
I added this because I noticed that the dropdown was showing below the hint, which looked odd.